### PR TITLE
Add integration test

### PR DESCRIPTION
**Pull request by kellysun878**
basic_test.go will run the cli command for each distro, determining
which distros can download and boot successfully.

Signed-off-by: Kelly Sun <sunkelly888@gmail.com>

### CODE REVIEW

File: cmds/cli/main.go [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+	"flag"
+	"fmt"
+	"html/template"
+	"io/ioutil"
+	"log"
+	"os"
+	"path"
+	"path/filepath"
+
+	Boot "github.com/u-root/u-root/pkg/boot"
```
Comment (rjoleary) [inline cmds/cli/main.go:16]: User local case import name:

```suggestion
	"github.com/u-root/u-root/pkg/boot"
```

File: cmds/cli/main.go [hunk 2]
```diff
@@ -0,0 +1,176 @@
+package main
```
Comment (rjoleary) [inline cmds/cli/main.go:1]: Add a package-level comment.

File: cmds/cli/main.go [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+		parsedConfigs, err := bootiso.ParseConfigFromISO(i.path, distro.BootConfig)
+		if err != nil {
+			return err
+		}
+
+		configs = append(configs, parsedConfigs...)
+	}
+
+	if len(distro.CustomConfigs) != 0 {
+		CustomConfigs, err := bootiso.LoadCustomConfigs(i.path, distro.CustomConfigs)
```
Comment (rjoleary) [inline cmds/cli/main.go:54]: Use lower-case:

```suggestion
		customConfigs, err := bootiso.LoadCustomConfigs(i.path, distro.CustomConfigs)
```

File: cmds/cli/main.go [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+	err = json.Unmarshal([]byte(data), &supportedDistros)
+	if err != nil {
+		return fmt.Errorf("Could not unmarshal JSON file: %v\n", err)
+	}
+
+	return nil
+}
+
+func main() {
+
```
Comment (rjoleary) [inline cmds/cli/main.go:150]: Remove empty line.

File: integration/basic_test.go [hunk 5]
```diff
@@ -18,19 +19,30 @@ func TestScript(t *testing.T) {
...
+				"github.com/u-root/u-root/cmds/core/ip",
+				"github.com/u-root/u-root/cmds/core/shutdown",
+				"github.com/u-root/u-root/cmds/core/sleep",
+				"github.com/u-root/u-root/cmds/boot/pxeboot",
+				"github.com/u-root/webboot/cmds/webboot",
+				"github.com/u-root/webboot/cmds/cli",
+			),
 			ExtraFiles: []string{
-				"./testdata/distros.json",
+				// "./testdata/distros.json",
```
Comment (pgherghetta) [inline integration/basic_test.go:33]: Remove this comment if it's not needed.

**Review by pgherghetta — APPROVED** [review]

**Review by pgherghetta — APPROVED** [review]

**Pull request merged**